packages/docusaurus/src/server/plugins/configs.ts

```diff
@@ -120,7 +120,7 @@ export async function loadPluginConfigs(
     // Site config should be the highest priority.
     ...standalonePlugins,
     ...standaloneThemes,
-  ];
+  ].filter(<T>(x: T | false | null | undefined): x is T => Boolean(x));
```

Comment:
That's a change I think we could do and merge in another PR asap

I think we have an issue for it btw or it was discussed in the past